### ISSUE TITLE
Not downloading media from "LikedBy" Tumblr URLs 

### ISSUE THREAD

**Issue by arrowsmith001** [TumblThreeApp/TumblThree#136]
**Describe the bug**
When trying to crawl with a "liked by" URL (i.e. https://www.tumblr.com/liked/by/tania-rivilis) the folder is created but is empty.

**To Reproduce**
1. Create new TumblThree project
2. Build solution and run
3. Find a tumblr page of the form https://www.tumblr.com/liked/by/{Blog Name}, making sure that it comes up in a web browser without saying "Permission Denied"
3. Add url to "Add Blog", queue and crawl

**Expected behavior**
Media from the page is downloaded.

**Desktop (please complete the following information):**
 - TumblThree version: 1.4.0.0
 - OS: Windows 10 Pro

**Comment by thomas694** [comment]
It works without a problem here, even with this blog.
Maybe you have to give the privacy consent (new button in the settings) again or even do a re-login.

**Comment by arrowsmith001** [comment]
I didn't even know these buttons existed within the app! I don't know how it was working before. Maybe it didn't need "me" to be logged in to just pull from blogs in the regular way. Anyway it's downloading stuff now. Thanks.

**Comment by thomas694** [comment]
Formerly they used a relative easy privacy consent mechanism. We just had to call an url and grab some cookies. But a while ago they changed it and now use a way more complicated one. Several JS, multiple calls and cookies involved. We would have to analyze and mimic their way, every time they adapt it. So we decided to use CEF (embedded chromium), let the browser handle it and just copy the final cookies. The authenticate button was born.
Recently they introduced another privacy consent mechanism for the Tag/LikedBy searches. So the second button was born.
